dids/src/main/kotlin/web5/sdk/dids/DidIon.kt

```diff
@@ -70,43 +73,41 @@ public class DidIonConfiguration internal constructor(
 
 
 /**
- * Returns a [DidIonManager] after applying the provided configuration [builderAction].
+ * Returns a [DidIonApi] after applying the provided configuration [builderAction].
  */
-public fun DidIonManager(builderAction: DidIonConfiguration.() -> Unit): DidIonManager {
+public fun DidIonApi(builderAction: DidIonConfiguration.() -> Unit): DidIonApi {
   val conf = DidIonConfiguration().apply(builderAction)
-  return DidIonManagerImpl(conf)
+  return DidIonApiImpl(conf)
 }
 
-/** [DidIonManager] is sealed, so we provide an impl so the constructor can be called. */
-private class DidIonManagerImpl(configuration: DidIonConfiguration) : DidIonManager(configuration)
+/** [DidIonApi] is sealed, so we provide an impl so the constructor can be called. */
+private class DidIonApiImpl(configuration: DidIonConfiguration) : DidIonApi(configuration)
```

Comment:
Good question. First, let me get at the motivation. I wanted the users of the SDK to experience the following behaviors:

1. Able to call `create` like a static method

```kt
val did = DidIonApi.create()
```

2. Able to override some defaults with the builder pattern.

```kt
val api = DidIonApi {
  ionHost = "https://mycustomionhost.com"
}
val did = api.create()
```

3. Disable calling constructors for `DidIonApi`, so it can only be created with the builder pattern in 2. The motivation behind this is to remove the optionality on how to create `DidIonApi` objects. 


For 1, you need to have some sort of companion object.

For 2, you need something like 

```kt
public fun DidIonApi(configurationBlock: DidIonConfiguration.() -> Unit): DidIonApi
```

No problem so far. But for 3, that's where this kicks in. As you correctly pointed out, removing the `sealed` keyword would allow users of the SDK to call `DidIonApi(configuration)`.